frontend/src/features/org/org-status-banner.ts

```diff
@@ -138,21 +144,39 @@ export class OrgStatusBanner extends BtrixElement {
                       str`You have ${daysDiff} days left of your Browsertrix trial`,
                     ),
 
-            detail: html`
-              <p>
-                ${msg(
-                  html`Your free trial ends on ${dateStr}. To continue using
-                    Browsertrix, select <strong>Subscribe Now</strong> in
-                    ${billingTabLink}.`,
-                )}
+            detail: html`<p>
+                ${msg(str`Your free trial ends on ${dateStr}.`)}
+                ${isCancelingTrial
+                  ? msg(
+                      html`To continue using Browsertrix, select
+                        <strong>Subscribe Now</strong> in ${billingTabLink}.`,
+                    )
+                  : subscription?.futureCancelDate
+                    ? // TODO See if we can differentiate whether the trial will rollover
+                      // (card on file) or become read-only because no card on file
```

Comment:
@emma-sg @ikreymer is this straightforward to implement? I can create an issue if so.